packages/flutter/test/material/expansion_tile_initial_state_test.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter/material.dart';
+
+void main() {
+    testWidgets('ExpansionTile initial state', (WidgetTester tester) async {
+    final Key topKey = new UniqueKey();
+    final Key expandedKey = const PageStorageKey<String>('expanded');
+    final Key collapsedKey = const PageStorageKey<String>('collapsed');
+    final Key defaultKey = const PageStorageKey<String>('default');
+
+    final Key tileKey = new UniqueKey();
+
+    final Map<String, WidgetBuilder> routes = <String, WidgetBuilder>{
+      '/': (_) {
+        return new Material(
+          child: new SingleChildScrollView(
+            child: new Column(
+              children: <Widget>[
+                new ListTile(title: const Text('Top'), key: topKey),
+                new ExpansionTile(
+                  key: expandedKey,
+                  initiallyExpanded: true,
+                  title: const Text('Expanded'),
+                  children: <Widget>[
+                    new ListTile(
+                      key: tileKey,
+                      title: const Text('0')
+                    ),
+                  ]
+                ),
+                new ExpansionTile(
+                  key: collapsedKey,
+                  initiallyExpanded: false,
+                  title: const Text('Collapsed'),
+                  children: <Widget>[
+                    new ListTile(
+                      key: tileKey,
+                      title: const Text('0')
+                    ),
+                  ]
+                ),
+                new ExpansionTile(
+                  key: defaultKey,
+                  title: const Text('Default'),
+                  children: <Widget>[
+                    const ListTile(title: const Text('0')),
+                  ]
+                ),
+              ]
+            )
+          )
+        );
+      }
+    };
+
+    await tester.pumpWidget(new MaterialApp(routes: routes));
```

Comment:
Using `home:` instead of `routes:` would simplify this test

Reply:
Done